src/components/tabs/themes/shared/tab/tab.fluent.scss

```diff
@@ -3,14 +3,28 @@
 
 $theme: $fluent;
 
-:host {
-    ::slotted(igc-icon) {
-        color: var-get($theme, 'item-icon-color');
-    }
+[part='tab-header'] {
+    transition: all .3s cubic-bezier(.35, 0, .25, 1);
```

Comment:
I see the timing function declared in several places. Questioning if this is synced with the implementation in Ignite UI for Angular, and whether we can use one of the included timing functions from the theming package. Else, I would prefer that this value is synced between the two libraries and declared only once, in the common file for instance.

Reply:
We don't have that exact bezier function. The closest one is $in-sine: cubic-bezier(0.47, 0, 0.745, 0.715). Let me know if you'd like me to update it. I’d prefer to handle it in a separate PR after this one is merged, as it will also require updating Angular.